homeworks/simonlaszlo/3_conditionals_loops/flat_finder.py

```diff
@@ -0,0 +1,16 @@
+cityes=("New York","San Francisco","Washington","Chichago")
+city_name,city_cost=input("Please enter the city name and the monthly fee of the sublet, separated by commas! ").split(",")
+city_name=city_name.strip().title()
+city_cost=int(city_cost)
+print(f"The City name is: {city_name}, monthly price: {city_cost}")
+if (city_name==cityes[0]  or city_name==cityes[1]) and city_cost<4000 :
```

Comment:
szintén nem hiba, csak egy egyszerűsítési lehetőség:
`if city_name in cities[0:2] and city_cost<4000:`